optuna/visualization/matplotlib/_pareto_front.py

```diff
@@ -155,6 +174,15 @@ def _get_pareto_front_3d(info: _ParetoFrontInfo) -> "Axes":
     ax.set_ylabel(info.target_names[info.axis_order[1]])
     ax.set_zlabel(info.target_names[info.axis_order[2]])
 
+    if info.infeasible_trials_with_values is not None and len(info.infeasible_trials_with_values) > 0:
+        ax.scatter(
+            xs=[values[info.axis_order[0]] for _, values in info.infeasible_trials_with_values],
+            ys=[values[info.axis_order[1]] for _, values in info.infeasible_trials_with_values],
+            zs=[values[info.axis_order[2]] for _, values in info.infeasible_trials_with_values],
+            color=cmap(1),
+            label="Infeasible",
```

Comment:
A nit, but could you add `Trial` like `Best Trial`
```suggestion
            label="Infeasible Trial",
```